### PR TITLE
Release v2.6

### PR DESCRIPTION
**Pull request by JieXiong9119**
#### What does this PR do?
Release BuildingSync 2.6

### CODE REVIEW

File: BuildingSync.xsd [hunk 1]
```diff
@@ -16110,6 +16139,66 @@
...
+      <xs:simpleContent>
+        <xs:extension base="xs:decimal">
+          <xs:attribute ref="auc:Source"/>
+        </xs:extension>
+      </xs:simpleContent>
+    </xs:complexType>
+  </xs:element>
+  <xs:element name="LifeCycleSavingsEnergyCost">
+    <xs:annotation>
+      <xs:documentation>Estimated present value of total cost savings in energy costs in the whole life cycle ($)</xs:documentation>
```
Comment (ThibaultMarzullo) [inline BuildingSync.xsd:16156]:
```suggestion
      <xs:documentation>Estimated present value of total savings in energy costs in the whole life cycle ($)</xs:documentation>
```

File: BuildingSync.xsd [hunk 2]
```diff
@@ -16110,6 +16139,66 @@
...
+      <xs:simpleContent>
+        <xs:extension base="xs:decimal">
+          <xs:attribute ref="auc:Source"/>
+        </xs:extension>
+      </xs:simpleContent>
+    </xs:complexType>
+  </xs:element>
+  <xs:element name="LifeCycleSavingsWaterCost">
+    <xs:annotation>
+      <xs:documentation>Estimated present value of total cost savings in water costs in the whole life cycle ($)</xs:documentation>
```
Comment (ThibaultMarzullo) [inline BuildingSync.xsd:16180]:
```suggestion
      <xs:documentation>Estimated present value of total savings in water costs in the whole life cycle ($)</xs:documentation>
```

File: BuildingSync.xsd [hunk 3]
```diff
@@ -16110,6 +16139,66 @@
...
+      <xs:simpleContent>
+        <xs:extension base="xs:decimal">
+          <xs:attribute ref="auc:Source"/>
+        </xs:extension>
+      </xs:simpleContent>
+    </xs:complexType>
+  </xs:element>
+  <xs:element name="LifeCycleSavingsAncillaryCost">
+    <xs:annotation>
+      <xs:documentation>Estimated present value of total cost savings in ancillary/other costs in the whole life cycle ($)</xs:documentation>
```
Comment (ThibaultMarzullo) [inline BuildingSync.xsd:16192]:
```suggestion
      <xs:documentation>Estimated present value of total savings in ancillary/other costs in the whole life cycle ($)</xs:documentation>
```

File: proposals/2024/Add LifeCycleSavings elements.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
+# Add LifeCycleSavings elements
+
+## Overview
+
+This proposal is to add new elements related to Life Cycle savings data for BuildingSync-CTS mapping:
```
Comment (ThibaultMarzullo) [inline proposals/2024/Add LifeCycleSavings elements.md:5]:
```suggestion
This proposal is to add new elements related to Life Cycle savings data for mapping FEMP's Compliance Tracking System fields to BuildingSync Schema: 
```

File: proposals/2024/Add LifeCycleSavings elements.md [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+      <xs:simpleContent>
+        <xs:extension base="xs:decimal">
+          <xs:attribute ref="auc:Source"/>
+        </xs:extension>
+      </xs:simpleContent>
+    </xs:complexType>
+  </xs:element>
+  <xs:element name="LifeCycleSavingsEnergyCost">
+    <xs:annotation>
+      <xs:documentation>Estimated present value of total cost savings in energy costs in the whole life cycle ($)</xs:documentation>
```
Comment (ThibaultMarzullo) [inline proposals/2024/Add LifeCycleSavings elements.md:46]:
```suggestion
      <xs:documentation>Estimated present value of total savings in energy costs in the whole life cycle ($)</xs:documentation>
```

File: proposals/2024/Add LifeCycleSavings elements.md [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+      <xs:simpleContent>
+        <xs:extension base="xs:decimal">
+          <xs:attribute ref="auc:Source"/>
+        </xs:extension>
+      </xs:simpleContent>
+    </xs:complexType>
+  </xs:element>
+  <xs:element name="LifeCycleSavingsWaterCost">
+    <xs:annotation>
+      <xs:documentation>Estimated present value of total cost savings in water costs in the whole life cycle ($)</xs:documentation>
```
Comment (ThibaultMarzullo) [inline proposals/2024/Add LifeCycleSavings elements.md:70]:
```suggestion
      <xs:documentation>Estimated present value of total savings in water costs in the whole life cycle ($)</xs:documentation>
```

File: proposals/2024/Add LifeCycleSavings elements.md [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+      <xs:simpleContent>
+        <xs:extension base="xs:decimal">
+          <xs:attribute ref="auc:Source"/>
+        </xs:extension>
+      </xs:simpleContent>
+    </xs:complexType>
+  </xs:element>
+  <xs:element name="LifeCycleSavingsAncillaryCost">
+    <xs:annotation>
+      <xs:documentation>Estimated present value of total cost savings in ancillary/other costs in the whole life cycle ($)</xs:documentation>
```
Comment (ThibaultMarzullo) [inline proposals/2024/Add LifeCycleSavings elements.md:82]:
```suggestion
      <xs:documentation>Estimated present value of total savings in ancillary/other costs in the whole life cycle ($)</xs:documentation>
```

**Review by ThibaultMarzullo — CHANGES_REQUESTED** [review]
Great job! I have suggested a few changes

**Comment by JieXiong9119** [comment]
> Great job! I have suggested a few changes

Updated. Please review again~

**Pull request merged**